### PR TITLE
FS-4286: application store sqs extended client changes to send messages to notification service queue

### PR DESCRIPTION
**Pull request by nuwan-samarasinghe**
https://dluhcdigital.atlassian.net/browse/FS-4286

### Change description
application store sqs extended client changes to send messages to notification service queue

- [x] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
After completing or if there are applications not completed message. will be sent and it should successfully represent on notify service

### CODE REVIEW

File: api/routes/queues/routes.py [hunk 1]
```diff
@@ -16,24 +20,44 @@ def post_submitted_application_to_assessment(self, application_id=None):
...
-            message_submitted_id = _SQS_CLIENT.submit_single_message(
-                queue_url=Config.AWS_SQS_IMPORT_APP_PRIMARY_QUEUE_URL,
-                message=application_with_form_json,
-                extra_attributes=application_attributes,
-                message_group_id="import_applications_group",
-                message_deduplication_id=str(uuid4()),  # ensures message uniqueness
-            )
-
-            return f"Message queued, message_id is: {message_submitted_id}.", 201
+            """
```
Comment (srh-sloan) [inline api/routes/queues/routes.py:29]: This code seems to do the same thing as https://github.com/communitiesuk/funding-service-design-application-store/pull/343/files#diff-92de12e32db2cfd608043c63a25bbd576f0cafb49cb55d8d46f747270936a080R252 above, is that right?

**Review by srh-sloan — CHANGES_REQUESTED** [review]
Generally looking good, a couple of small specific comments.
As a general point though can we rename anything that has assessment in it please? Application store doesn't need to know about assessment, really it should be putting stuff on a 'submission' queue as what happens to it after that this service doesn't need to know or care about.

File: api/routes/application/routes.py [hunk 2]
```diff
@@ -249,10 +240,42 @@ def submit(self, application_id):
...
 
+    def _send_assessment_queue(self, application_id, application_with_form_json):
+        application_attributes = {
+            "application_id": {"StringValue": application_id, "DataType": "String"},
+            "S3Key": {
+                "StringValue": ASSESSMENT_S3_KEY_CONST,
+                "DataType": "String",
+            },
+        }
+        # Submit message to queue, in a future state this can trigger the
```
[the remark this answers is not in the document]
Comment (nuwan-samarasinghe) [reply api/routes/application/routes.py]: Fixed

File: api/routes/application/routes.py [hunk 3]
```diff
@@ -45,6 +46,9 @@
 from fsd_utils.config.notify_constants import NotifyConstants
 from sqlalchemy.orm.exc import NoResultFound
 
+ASSESSMENT = "import_applications_group"
```
[the remark this answers is not in the document]
Comment (nuwan-samarasinghe) [reply api/routes/application/routes.py]: Fixed

File: api/routes/queues/routes.py [hunk 4]
```diff
@@ -1,11 +1,15 @@
...
+import json
 from uuid import uuid4
 
 from config import Config
 from db.queries import get_application
-from external_services.aws import _SQS_CLIENT
+from flask import current_app
 from flask.views import MethodView
 
+ASSESSMENT = "import_applications_group"
```
[the remark this answers is not in the document]
Comment (nuwan-samarasinghe) [reply api/routes/queues/routes.py]: Fixed

**Review by srh-sloan — APPROVED** [review]

**Pull request merged**